lib/src/component_declaration/component_base.dart

```diff
@@ -200,7 +245,7 @@ abstract class UiComponent<TProps extends UiProps> extends react.Component imple
 
   /// A typed props object corresponding to the current untyped props Map ([unwrappedProps]).
   ///
-  /// Created using [typedPropsFactory] and cached for each Map instance.
+  /// Created using [typedPropsFactory] and cached for each Map instance within [_typedPropsCache].
```

Comment:
#nit `within [_typedPropsCache]` is an implementation detail not relevant to consumers of this API (and is also evident by looking at the code).